src/Tribe/Admin/Notice/Service_Provider.php

```diff
@@ -54,24 +52,5 @@ public function plugins_loaded() {
 		tribe( 'pue.notices' );
 		tribe( 'admin.notice.php.version' );
 		tribe( WP_Version::class );
-
-		if ( defined( 'TRIBE_HIDE_MARKETING_NOTICES' ) ) {
```

Comment:
You moved the notice instantiation to a controller, but you've removed this check in the process...we had this check for specific reasons (like network installs) - is that intentional?

Reply:
No I didnt, that one was legacy. We moved to `tec_should_hide_upsell()`